oShare/MPC/MultipeerConnectivityManager.swift

```diff
@@ -86,11 +100,7 @@ extension MultipeerConnectivityManager: MCSessionDelegate {
 		case .connecting:
 			// TODO: - Display Indicator?
 			break
-		default:
-			startAdvertising()
-			startBrowsingForDevices()
-
-			delegate?.failedToConnect(withPeer: peerID)
```

Comment:
This was removed because it presented the _failure to connect_ error box when you _wanted_ to disconnect the session.